yoti/com_yoti.xml

```diff
@@ -1,34 +1,44 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <extension type="component" version="3.0" method="upgrade">
-    <name>Yoti Connect</name>
-    <author>Simon Tong</author>
-    <authorEmail>simon.tong@yoti.com</authorEmail>
+    <name>com_yoti</name>
+    <author>Yoti Ltd</author>
+    <authorEmail>sdksupport@yoti.com</authorEmail>
     <authorUrl>http://www.yoti.com</authorUrl>
     <creationDate>August 2016</creationDate>
-    <version>1.0.0</version>
+    <version>1.0.1</version>
     <license>Link to licence url</license>
     <copyright>Copyright (C) 2016 http://www.yoti.com. All rights reserved.</copyright>
```

Comment:
2016-2017?

Reply:
Update: 2016 should be updated to 2017 everywhere. We will update these values yearly.

Reply:
Yes I thought about hiding Yoti profile on edit profile page, but also thought it could be confusing to users when they can see Yoti data on profile display, and the data disappear when they edit their profile. I wouldn't mind hiding it, but their could be some confusion there.